code/modules/hydroponics/seed.dm

```diff
@@ -457,7 +457,8 @@
 			/decl/reagent/water,
 			/decl/reagent/woodpulp,
 			/decl/reagent/ambrosia_extract,
-			/decl/reagent/skrell_nootropic
+			/decl/reagent/skrell_nootropic,
+			/decl/reagent/toxin/berserk
```

Comment:
If I understand the ``seed.dm`` file right, this will mean that random vines/plants will have a chance to get Red Nightshade in their spawn and mutation pool? If so, you might want to mention it in the changelog.

Otherwise, it looks great! I was originally a bit concerned about the damage increase, but upon reading the changes further I think it should be alright since it won't be as easy to make, and it'll still knock the liver flat like an indebted person getting visited by IRUs.

Reply:
Great work dude!